ConcernsCaseWork/ConcernsCaseWork.CypressTests/cypress/pages/createCase/createCaseDetailsPage.js

```diff
@@ -130,8 +130,17 @@ class CreateCaseDetailsPage {
         cy.get('#next-steps-info').then(($nxtinf1) =>{
             expect($nxtinf1).to.be.visible
             expect($nxtinf1.text()).to.match(/(4000 characters)/i)
+        });
+        
+        //Case history validation
+        cy.get('[class="govuk-grid-row"] *[for="case-history"]').should(($nxt) => {
+            expect($nxt.text().trim()).equal("Next steps (optional)")
```

Comment:
I just want to check - is this correct that it says 'Next steps..'? when it appears to be for the case history text box